ui/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+    "extends": ["eslint:recommended", "plugin:react/recommended"],
+    rules: {
+        // TODO turn these recommended rules back on
```

Comment:
Should this be addressed here?

Reply:
rules we don't already follow have been turned off. this TODO is a reminder to some day fix all the violations and turn the rules on

Reply:
I recommend updating the TODO with that info